.github/workflows/compile-examples.yml

```
@@ -0,0 +1,265 @@
+      - name: Checkout ArduinoCore-API
+        uses: actions/checkout@v2
+        with:
+          repository: arduino/ArduinoCore-API
+          path: extras/ArduinoCore-API
+
+      - name: Install ArduinoCore-API
+        run: mv "$GITHUB_WORKSPACE/extras/ArduinoCore-API/api" "$GITHUB_WORKSPACE/cores/arduino"
+
+      - name: Checkout Basic examples
+        uses: actions/checkout@v2
+        with:
+          repository: arduino/arduino-examples
+          path: extras
+
+      - name: Compile examples
+        uses: arduino/compile-sketches@main
+        with:
+          fqbn: ${{ matrix.board.fqbn }}
+          libraries: |
+            - source-url: https://github.com/giulcioffi/WiFi101.git
+              version: namespaced_api
+            - name: MFRC522
+            - name: Arduino_MKRMEM
+            - name: FlashStorage
+            - source-url: https://github.com/arduino-libraries/Keyboard.git
+            - name: Mouse
+            - name: Servo
+            - name: LiquidCrystal
+            - name: CapacitiveSensor
+            - name: Ethernet
+            - name: ArduinoBearSSL
+            - name: Arduino_APDS9960
+            - name: Arduino_LSM9DS1
+            - name: ArduinoHttpClient
+            - name: NTPClient
+            - name: TFT
+            - name: ArduinoMqttClient
+            - name: Arduino_CRC32
+            - name: Arduino_LSM6DS3
+            - name: Stepper
+            - name: SD
+            - name: Arduino_JSON
+            - name: Arduino_HTS221
+            - name: Firmata
+            - name: ArduinoMotorCarrier
+            - name: ArduinoCloudThing
+            - name: Arduino_DebugUtils
+            - name: WiFi Link
+            - name: Arduino_LPS22HB
+            - name: CTC GO MOTIONS
+            - name: ArduinoModbus
+            - name: ArduinoIoTCloudBearSSL
+            - name: ArduinoDMX
+            - name: ArduinoRS485
+            - name: Arduino_OAuth
+            - name: CTC GO CORE
+            - name: WiFi
+            - name: Bridge
+            - name: Temboo
+            - name: ArduinoIoTCloud
+            - name: Madgwick
+            - name: MKRGSM
+            - name: MKRNB
+            - name: LoRa
+            - name: MKRWAN
+            - name: WiFiNINA
+          platforms: |
+            # Use Board Manager to install the latest release of Arduino megaAVR Boards to get the toolchain
+            - name: "arduino:samd"
+            # Overwrite the Board Manager installation with the local platform
+            - source-path: "./"
+              name: "arduino:samd"
+          sketch-paths: |
+            ${{ env.UNIVERSAL_SKETCH_PATHS }}
+            ${{ matrix.additional-sketch-paths }}
+          enable-deltas-report: 'true'
+          verbose: 'true'
+
+      - name: Save memory usage change report as artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: size-deltas-reports
+          path: size-deltas-reports
```

```suggestion
          name: sketches-reports
          path: sketches-reports
```
This change is required due to a recent breaking change to the `arduino/compile-sketches` action's default sketches report path and to the `arduino/report-size-deltas` action's default workflow artifact name:
https://github.com/arduino/compile-sketches/pull/7
https://github.com/arduino/report-size-deltas/pull/7